app/controllers/reports/regions_controller.rb

```diff
@@ -36,6 +41,15 @@ def details
     @quarterly_registrations = @data[:quarterly_registrations]
     @top_region_benchmarks = @data[:top_region_benchmarks]
     @last_registration_value = @data[:registrations].values&.last || 0
+
+    unless @region.is_a?(FacilityGroup)
+      @recent_blood_pressures = @region
```

Comment:
Wondering if this will read neater if we move this `recent_blood_pressures` method to the `Facility` model itself (and other regions subsequently as necessary) since it's being used in multiple places now.

So then, we could just do `@region.recent_blood_pressures`.